drivers/dp/Kconfig

```diff
@@ -28,4 +28,10 @@ config SWDP_BITBANG_DRIVER
 	help
 	  Serial Wire Debug Port bit-bang driver.
 
+config SWDP_MAINTAIN_RESET_DEASSERTED
+	bool "Serial Wire Debug Port keeps RESET deasserted even when inactive"
+	depends on SWDP_BITBANG_DRIVER
+	help
+	  Serial Wire Debug Port keeps RESET deasserted even when inactive
```

Comment:
Applying it to all instances of this driver does not seem like a good approach to me. Maybe it could be a DT property.